### PR TITLE
Move files from metype/tc-players-vue3 to this branch for PR

### PR DESCRIPTION
**Pull request by metype**
This is a new version of the site built atop Vue3 + Vuetify with Vite
Most functionality is the same; however, the player search results have been improved by now showing faces. Some other minor changes include a slightly nicer looking player info section and removal of the home button.

Any files that seem to be leftover from creating a new project, or just leftover in general can be pointed out but I think I've mostly cleaned it up! :D

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,45 +1,44 @@
...
+        <v-container fluid>
+          <router-view></router-view>
+        </v-container>
+      </v-main>
+      <v-toolbar color="primary" dark
+      class="position-fixed w-100 pa-0">
+        <!--          <v-toolbar-title id="title"  class="mr-3 ml-3" >-->
+        <!--            <v-icon>mdi-home</v-icon>-->
+        <!--          </v-toolbar-title>-->
+        <!--        </router-link>-->
```
Comment (Trikolon) [inline src/App.vue]: should this be removed?

File: src/main.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import {registerPlugins} from "./plugins";
+
+// const mq = window.matchMedia?.('(prefers-color-scheme: dark)');
+//
+// vuetify.theme.global.name.value = mq?.matches ? "dark" : "light";
+//
+// // eslint-disable-next-line no-unused-expressions
+// mq?.addEventListener?.('change', (e) => {
+//   vuetify.theme.global.name.value = e.matches ? "dark" : "light";
+// });
```
Comment (Trikolon) [inline src/main.ts]: Let's not leave unused code in here. If you want to keep track of something I recommend to file an issue.

File: src/main.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+// Hopefully temporary, but I have some hardcoded colors in the UI that
+// provide rather bad contrast in light mode and the simple fix is to disable
+// light mode until I figure out an easy way to make those colors dynamic
+// and always-readable
```
Comment (Trikolon) [inline src/main.ts]: Please file an issue and link it here

File: src/pages/index.vue [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <HelloWorld />
+</template>
+
+<script lang="ts" setup>
+  //
```
Comment (Trikolon) [inline src/pages/index.vue]: empty line?

File: src/styles/settings.scss [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/**
+ * src/styles/settings.scss
+ *
+ * Configures SASS variables and Vuetify overwrites
+ */
+
+// https://vuetifyjs.com/features/sass-variables/`
+// @use 'vuetify/settings' with (
+//   $color-pack: false
+// );
```
Comment (Trikolon) [inline src/styles/settings.scss:10]: Do we need this file if everything is commented out anyway?

**Review by Trikolon — DISMISSED** [review]
I only looked at this very broadly so don't consider it an in-depth review. Looks good overall.

File: src/pages/index.vue (hunk 4 above)
Comment (metype) [reply src/pages/index.vue]: The entire pages folder was leftover that didn't hurt anything so I overlooked it before making the PR, has been removed.

File: src/styles/settings.scss (hunk 5 above)
Comment (metype) [reply src/styles/settings.scss:10]: Unfortunately we do, this is referenced directly by Vuetify and cannot be removed as far as I can tell, even if it doesn't actually contain any overrides.

**Review by Trikolon — APPROVED** [review]

**Pull request merged**